src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,8 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static int ClimberDoubleSolenoied1 = 1;
+    public static int ClimberDoubleSolenoied2 = 2;
+    public static int ClimberTalonSRX1 = 1;
+    public static int ClimberTalonSRX2 = 2;
```

Comment:
Please have more specific and descriptive constant naming.
Perhaps use `CONSTANT_CASE`?
Don't worry, I'll go over Constant naming next meeting and explain constant naming.